example/lib/pages/multi_assets_page.dart

```diff
@@ -191,6 +191,23 @@ class _MultiAssetsPageState extends State<MultiAssetsPage> {
             );
           },
         ),
+        PickMethodModel(
```

Comment:
We should definitely make these methods factories. I'll migrate them later.

Reply:
You are right. On my next PR :-)